src/ipc/ipc3/helper.c

```diff
@@ -146,8 +146,8 @@ static const struct comp_driver *get_drv(struct sof_ipc_comp *comp)
 
 out:
 	if (drv)
-		tr_dbg(&comp_tr, "get_drv(), found driver type %d, uuid %pU",
-		       drv->type, drv->tctx->uuid_p);
+		tr_warn(&comp_tr, "get_drv(), found driver type %d msg %d, uuid %pU",
```

Comment:
this is the right execution code, drv is expected to be true, so this print will always happen, I would not suggest to print this or change it warning, keep as tr_dbg is better.

If delete this, may also make sense, due to if error happens, line 137 - 143 would print out message.

Reply:
Oh, that an unrelated debugging change that slipped in. I'll get rid of it.